backend/auth/handlers_user_account.go

```diff
@@ -0,0 +1,247 @@
+	rawUsers, err := dbadapter.WebuiDBClient.RestfulAPIGetMany(UserAccountDataColl, bson.M{})
+	if err != nil {
+		logger.DbLog.Errorln(err.Error())
+		return nil, errors.New(errorRetrieveUserAccounts)
+	}
+	var users []*configmodels.DBUserAccount
+	users = make([]*configmodels.DBUserAccount, 0)
+	for _, rawUser := range rawUsers {
+		var user configmodels.DBUserAccount
+		err := json.Unmarshal(configmodels.MapToByte(rawUser), &user)
+		if err != nil {
+			logger.DbLog.Errorf(errorRetrieveUserAccount)
+			continue
+		}
+		users = append(users, &user)
+	}
+	return users, nil
+}
+
+func GetUserAccount(c *gin.Context) {
+	logger.WebUILog.Infoln("get user account")
+	username := c.Param("username")
+	dbUserAccount, err := fetchDBUserAccount(username)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": errorRetrieveUserAccount})
+		return
+	}
+	if dbUserAccount == nil {
+		c.JSON(http.StatusNotFound, gin.H{"error": errorUsernameNotFound})
+		return
+	}
+	userResponse := configmodels.GetUserAccountResponse{
+		Username: dbUserAccount.Username,
+		Role:     dbUserAccount.Role,
+	}
+	c.JSON(http.StatusOK, userResponse)
+}
+
+func fetchDBUserAccount(username string) (*configmodels.DBUserAccount, error) {
+	filter := bson.M{"username": username}
+	rawUserAccount, err := dbadapter.WebuiDBClient.RestfulAPIGetOne(UserAccountDataColl, filter)
+	if err != nil {
+		logger.DbLog.Errorln(err.Error())
+		return nil, err
+	}
+	if len(rawUserAccount) == 0 {
+		return nil, nil
+	}
+	var userAccount configmodels.DBUserAccount
+	err = json.Unmarshal(configmodels.MapToByte(rawUserAccount), &userAccount)
+	if err != nil {
+		logger.AuthLog.Errorln(err.Error())
+		return nil, err
+	}
+	return &userAccount, nil
+}
+
+func CreateUserAccount(c *gin.Context) {
+	logger.WebUILog.Infoln("create user account")
+	var createUserParams configmodels.CreateUserAccountParams
+	err := c.ShouldBindJSON(&createUserParams)
+	if err != nil {
+		logger.AuthLog.Errorln(err.Error())
+		c.JSON(http.StatusBadRequest, gin.H{"error": errorInvalidDataProvided})
+		return
+	}
+	if createUserParams.Username == "" {
+		c.JSON(http.StatusBadRequest, gin.H{"error": errorMissingUsername})
+		return
+	}
+	if createUserParams.Password == "" {
+		c.JSON(http.StatusBadRequest, gin.H{"error": errorMissingPassword})
+		return
+	}
+	if !validatePassword(createUserParams.Password) {
+		c.JSON(http.StatusBadRequest, gin.H{"error": errorInvalidPassword})
+		return
+	}
+	newUserRole := UserRole
+	isFirstAccountIssued, err := IsFirstAccountIssued()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{"error": errorRetrieveUserAccounts})
+		return
+	}
+	if !isFirstAccountIssued {
+		newUserRole = AdminRole
+	}
+	dbUser, err := configmodels.CreateNewDBUserAccount(createUserParams.Username, createUserParams.Password, newUserRole)
+	if err != nil {
+		logger.AuthLog.Errorln(err.Error())
+		c.JSON(http.StatusInternalServerError, gin.H{"error": errorCreateUserAccount})
+	}
```

Comment:
```suggestion
		c.JSON(http.StatusInternalServerError, gin.H{"error": errorCreateUserAccount})
	return
	}
```
If there is an error, we can return here.